src/common/utils/keycloak.adapter.util.ts

```diff
@@ -87,20 +87,25 @@ async function createUserInKeyCloak(query, token) {
     },
     data: data,
   };
-
   let userResponse;
+  // try {
+  //   userResponse = await axios(config);
+  // } catch (e) {
+  //   console.log(e.response, "Keycloak Creation error");
```

Comment:
@Shubham4026 Please remove commented code

Reply:
Sure I have removed it. Will Send it next PR